packages/autoloader/src/autoload.php

```diff
@@ -94,6 +94,7 @@ function autoloader( $class_name ) {
 				return true;
 			}
 		}
+		return false;
```

Comment:
Why do we need this? For consistent method signature, or?

Reply:
I'm asking because the default `spl_autoload` doesn't return anything AFAIK.

Reply:
yeah, phpcs was mild complaining

Reply:
Could we `void` it?

Reply:
it needs to return `true` when it finds the class. so autoloading stops searching